feat(content): More new human planets and stations, update Kraken Station landing image (#9917)
endless-sky/endless-sky: data/map planets.txt

		threshold 5500
		fleet "Large Republic" 10

planet Cassandra
	attributes paradise uninhabited
	landscape land/mars4
	description `In the late 2300s, Cassandra was the subject of one of the most ambitious terraforming projects performed by humanity up to that point; an attempt to rapidly modify its biosphere and jumpstart the process of making the planet more than perfect for human habitation.`

> 	description `The project never came to be. For an as-yet-unexplained reason, some collapse or chain of destruction occurred during the terraforming project that made Cassandra uninhabitable, and ever since Cassandra has been discreetly left out of the public record by the terraforming industry.`

Reviewer: ```suggestion
	description `	The project never came to be. For an as-yet-unexplained reason, some collapse or chain of destruction occurred during the terraforming project that made Cassandra uninhabitable, and ever since Cassandra has been discreetly left out of the public record by the terraforming industry.`
```

		threshold 5500
		fleet "Large Republic" 10

planet Cassandra
	attributes paradise uninhabited
	landscape land/mars4

> 	description `In the late 2300s, Cassandra was the subject of one of the most ambitious terraforming projects performed by humanity until then; an attempt to rapidly modify its biosphere and jumpstart the process of making the planet more than perfect for human habitation.`

Reviewer 2: ```suggestion
	description `In the late 2300s, Cassandra was the subject of one of the most ambitious terraforming projects performed by humanity up to that point; an attempt to rapidly modify its biosphere and jumpstart the process of making the planet more than perfect for human habitation.`
```

Author: idk why this was never committed. Hold on.
```suggestion
	description `In the late 2300s, Cassandra was the subject of one of the most ambitious terraforming projects performed by humanity up to that point; an attempt to rapidly modify its biosphere and jumpstart the process of making the planet more than perfect for human habitation.`
```

Author: That's weird.

	description `	However, it was eventually realized after a series of fatal missteps that the project was far too difficult for the technology available then. An ecological collapse soon followed: much of Cassandra's wildlife went extinct, and its forests and oceans soon shriveled up into deserts and salt flats. The project was abandoned within a generation.`

> 	description `	While it taught valuable lessons to the studies of terraforming and planetary engineering then, Cassandra has largely been forgotten by humanity, overshadowed by the soon to be coming Turf Wars.`

Reviewer 2: ```suggestion
	description `	While it advanced the fields of terraforming and planetary engineering, Cassandra has largely been forgotten by humanity, overshadowed by the subsequent Turf Wars.`
```

Author: Sorry, changed the description :/

> 	"required reputation" -1000

Reviewer 3: ```suggestion
```

	bribe 0
	security 0.8

planet "Rowark's Roost"
	attributes moon uninhabited
	landscape land/moon1
	description "Rowark's Roost was once the hideout of one of the most famous privateers of the Southern Rim, Rowark Balois. From this remote moon he built an empire in slave and drug running, near-monopolizing the Rim during the early 27th century. The Roost itself was a stronghold of hangars and tunnels that would become the model for pirate cells across human space."
	description "In 2655, Rowark was flushed out of his Roost by a combined Rim and Deep fleet in a battle that continues to be studied by every aspirant Navy officer in their tutelage. As you fly over the moon you spy the wrecks of countless ships dotting its surface, but what is most striking is the truly impressive battleship-sized crater that marks the spot where Rowark met his end, doggedly choosing to fight to the very last."

> 

Reviewer 4: ```suggestion
	government Uninhabited

```
Unless there's anyone still doing things there or claiming ownership of it, it should probably use the Uninhabited government.

	bribe 0
	security 0.8

planet "Rowark's Roost"
	attributes moon uninhabited
	landscape land/moon1

> 	description `Rowark's Roost was once the hideout of one of the most famous privateers of the Southern Rim, Rowark Balois. From this remote moon he became one of the largest players in drug-running and slave trading in the Rim during the early 27th century. The extensive stronghold of hangars and tunnels he built here remain to this day.`

Author: Should be remains, yeah.
```suggestion
	description `Rowark's Roost was once the hideout of one of the most famous privateers of the Southern Rim, Rowark Balois. From this remote moon he became one of the largest players in drug-running and slave trading in the Rim during the early 27th century. The extensive stronghold of hangars and tunnels he built here remains to this day.`
```

> 	description "Rowark's Roost was once the hideout of one of the most famous privateers of the Southern Rim, Rowark Balois. From this remote moon he became one of the largest players in drug-running and slave trading in the Rim during the early 27th century. The extensive stronghold of hangars and tunnels he built here remain to this day."
> 	description "In 2655, Rowark was flushed out of his Roost by a combined Rim and Deep fleet in a battle that continues to be studied by every aspirant Navy officer in their tutelage. As you fly over the moon you spy the wrecks of countless ships dotting its surface, but what is most striking is the truly impressive battleship-sized crater that marks the spot where Rowark met his end, doggedly choosing to fight to the very last."

Reviewer: ```suggestion
	description `Rowark's Roost was once the hideout of one of the most famous privateers of the Southern Rim, Rowark Balois. From this remote moon he became one of the largest players in drug-running and slave trading in the Rim during the early 27th century. The extensive stronghold of hangars and tunnels he built here remain to this day.`
	description `	In 2655, Rowark was flushed out of his Roost by a combined Rim and Deep fleet in a battle that continues to be studied by every aspirant Navy officer in their tutelage. As you fly over the moon you spy the wrecks of countless ships dotting its surface, but what is most striking is the truly impressive battleship-sized crater that marks the spot where Rowark met his end, doggedly choosing to fight to the very last.`
```

	bribe 0.1
	security 0.5

planet Taylor
	landscape land/moon0
	attributes research core moon military

> 	description "Taylor is home to a large astronomical laboratory, with radio telescopes and other devices for studying the galactic core and the massive black hole that resides there. Because of the laboratory, Taylor has strict landing protocols that ensure your ship doesn't cross the path of the telescopes at any time."
> 	description "	On the other side of the moon the Syndicate has established a substantial military base, as a guard against the pirates in nearby Alcyone."
> 	description "	If you need to repair or refuel in this system, you will need to visit Sundive instead of this moon, as the military base is not open to ships that aren't affiliated with the Syndicate, and the laboratory's small spaceport does not have the facilities to service ships."

Reviewer: ```suggestion
	description `Taylor is home to a large astronomical laboratory, with radio telescopes and other devices for studying the galactic core and the massive black hole that resides there. Because of the laboratory, Taylor has strict landing protocols that ensure your ship doesn't cross the path of the telescopes at any time.`
	description `	On the other side of the moon the Syndicate has established a substantial military base, as a guard against the pirates in nearby Alcyone.`
	description `	If you need to repair or refuel in this system, you will need to visit Sundive instead of this moon, as the military base is not open to ships that aren't affiliated with the Syndicate, and the laboratory's small spaceport does not have the facilities to service ships.`
```
